pylsp_black/plugin.py

```diff
@@ -168,3 +174,13 @@ def load_config(filename: str) -> Dict:
     logger.info("Using config from %s: %r", pyproject_filename, config)
 
     return config
+
+
+def load_config(filename: str, client_config: Config) -> Dict:
+    settings = client_config.plugin_settings("black")
+
+    # Use the original, not cached function to load settings if requested
+    if not settings.get("cache_config", True):
```

Comment:
If we default to `True` users could get confused as to why their changes to black configuration files are not being observed (they would need to restart the LSP server). I think either we default to `False` or we document configuration caching in README. We should document plugin configuration anyway, so maybe we can go with the second option.

Reply:
I'm updating the README to document plugin configuration.

Reply:
Sorry, I thought you wanted this attribute to be `True`. I'll change it to `False` then.